packages/sprotty/package.json

```diff
@@ -34,8 +34,6 @@
   "devDependencies": {
     "@types/chai": "^4.3.6",
```

Comment:
@gfontorbe 
Could you also remove this dependency to `@types/chai` please? It is not needed I think.